### DISCUSSION TITLE
SubItems Group Controls

### DISCUSSION THREAD

**Discussion by chaimel** [fabricjs/fabric.js#9650] [Q&A] [answered]
Hi , Ive tried to active the group so i can edit the subitems like controlling the controls  but itried many methods still cannot control the controls of the objects inside a group

Im using v5 of the fabric for some reasons 

this event is using the mouse:down

var clickedObject = event.target;
      const target = event?.subTargets[0]
      if (clickedObject) {
        if (clickedObject.isType('group')) {
            clickedObject.set({
                angle : 0,
                scaleX: 1,
                scaleY: 1,
                editable: true,

                selectable: true,
            })
            if(target){
                target.hasControls = true;
                window.svgCanvas.setActiveObject(target)
                target.set({editable:true,borderColor :'red'});
                target.selectable = true;
                target.selection = true;
                target.setControlsVisibility({
                    tl: true, // Top-left control
                    tr: true, // Top-right control
                    br: true, // Bottom-right control
                    bl: true, // Bottom-left control
                    mt: true, // Middle-top control
                    mb: true, // Middle-bottom control
                    ml: true, // Middle-left control
                    mr: true, // Middle-right control
                    mtr: true, // Rotation control
                });
                
                target.setCoords();
                window.svgCanvas.requestRenderAll();

                if (event.e.detail === 2) {

                    var initText = target.text

                    if(target.type === "textbox"){
                        target.enterEditing()
                    }
                    else{
                        // target.setControlsVisibility({
                        //     tl: true,
                        //     tr: true,
                        //     br: true,
                        //     bl: true,
                        //     mt: true,
                        //     mb: true,
                        //     ml: true,
                        //     mr: true,
                        //     mtr: true,
                        // });
                    }
                    target.on('editing:exited', function () {
                        var new_text = target.text;
                        if(initText != new_text){
                            clickedObject.addWithUpdate();
                            window.svgCanvas.requestRenderAll();
                        }
                        else{
                            return;
                        }
                    });

                    // Manually update the controls after editing exited
                    target.on('scaling', function () {
                        // The 'scaling' event is triggered after the editing is exited
                        // Adjust controls position if needed
                        target.setCoords();
                    });

                } 
            }
          
        }else {
            console.log('Clicked on a single object:', clickedObject);
![1707098405261](https://github.com/fabricjs/fabric.js/assets/57937227/95af7ddc-9dbe-4677-9dd4-ed2d5dde00c7)

        }
![1707098301023](https://github.com/fabricjs/fabric.js/assets/57937227/187b91f4-3749-4afc-89ab-1db927aeb97f)

**Comment by ShaMan123** [comment, marked as answer]
2 things:
1.  tickets will be closed until a clear and concise **LIVE** reproduction is provided 
2. v5 group is a sink hole we do not support anymore #7670
